kong/tests/compose/docker-compose.yml

```diff
@@ -1,76 +1,91 @@
 version: '2.1'
```

Comment:
While we're here

```suggestion
version: '3'
```

Reply:
For the record, we can't do this because it uses health check in depends_on.